data/publications.html

```diff
@@ -42,4 +95,97 @@ <h2>All of Nir's Publications</h2>
         <p>&copy; 2024 Shavit Lab</p>
     </footer>
 </body>
-</html>
\ No newline at end of file
+</html>
+
+    
+    <p><strong>2018</strong></p>
```

Comment:
should these entries be in the unordered list above?

Reply:
Yeah, half of the formatting is still a WIP. I'm manually re-formatting and organizing all of the citations so it's taking a little longer than expected, so I just put a PR to mark progress and marked it as a draft.

Reply:
Ok, go ahead and request my review again when you finish the changes 👍